src/orgwarden/__main__.py

```diff
@@ -14,13 +14,30 @@
 app = typer.Typer(rich_markup_mode="markdown")
 
 
+def reject_empty_string(value: str):
+    if not value:
+        raise typer.BadParameter("Input cannot be an empty string.")
+    else:
+        return value
+
+
 @app.command()
 def list_repos(
     url: Annotated[
         str,
         typer.Argument(
             help="The url of a GitHub organization. Ex: 'https://github.com/gt-tech-ai'",
             show_default=False,
+            callback=reject_empty_string,
+        ),
+    ],
+    gh_pat: Annotated[
+        str,
+        typer.Argument(
+            help="A GitHub Personal Access Token (PAT) - must have access to the specified organization. "
+            "See [OrgWarden docs](https://github.com/gt-tech-ai/OrgWarden#setting-up-a-personal-access-token) for help setting up a PAT.",
+            show_default=False,
+            callback=reject_empty_string,
```

Comment:
This isn't something that you need to change, but rather a different perspective to consider...

I probably won't have introduced code to reject empty strings and would have instead let the GitHub API return an error if someone tried to provide an empty string. The reason is that there may be a number of different conditions that could be enforced (for example, max token length, ensuring that the token starts with "gh", etc.). 

Rather than writing code that I need to test and maintain, I would have relied on the GitHub API to validate its token and return the correct error message.